indexing-service/src/main/java/io/druid/indexing/common/task/IndexTask.java

```diff
@@ -151,7 +155,9 @@ public IndexTask(
       @JsonProperty("resource") TaskResource taskResource,
       @JsonProperty("spec") IndexIngestionSpec ingestionSchema,
       @JacksonInject ObjectMapper jsonMapper,
-      @JsonProperty("context") Map<String, Object> context
+      @JsonProperty("context") Map<String, Object> context,
+      @JacksonInject IndexMerger indexMerger,
+      @JacksonInject IndexIO indexIO
```

Comment:
@xvrl @gianm ok removed them from individual task constructors, added to TaskToolbox instead and using from same.
hopefully this will reduce some burden on tranquility side.